app/src/main/java/com/duckduckgo/app/browser/omnibar/OmnibarLayout.kt

```diff
@@ -495,7 +495,7 @@ open class OmnibarLayout @JvmOverloads constructor(
             }
 
             OmnibarLayoutViewModel.LeadingIconState.PRIVACY_SHIELD -> {
-                val isExperimentEnabled = appPersonalityFeature.self().isEnabled() && !appPersonalityFeature.variant1().isEnabled()
+                val isExperimentEnabled = !senseOfProtectionExperiment.isUserEnrolledInAVariantAndExperimentEnabled()
```

Comment:
This should be (remove the negation): 
val isExperimentEnabled = senseOfProtectionExperiment.isUserEnrolledInAVariantAndExperimentEnabled()